source/plugin/Assets/GoogleMobileAds/Common/DummyClient.cs

```diff
@@ -76,7 +76,8 @@ public void Initialize(string appId)
         public void Initialize(Action<IInitializationStatusClient> initCompleteAction)
         {
             Debug.Log("Dummy " + MethodBase.GetCurrentMethod().Name);
-            initCompleteAction(null);
+            var initStatusClient = new GoogleMobileAds.Unity.InitializationStatusClient();
```

Comment:
add 'using GoogleMobileAds.Unity;' at top?

Reply:
since you've added that you can change this line to:

`var initStatusClient = new InitializationStatusClient();`